app/manifest.webapp

```diff
@@ -3,7 +3,6 @@
   "description": "Tool for editing apps on-device",
   "customizations": [{
     "filter": ".*",
-    "css": ["style/app.css"],
```

Comment:
Would be better if we could comment this out, rather than removing it entirely. I'm not sure how to do that, though.